client/src/components/ArtistSidebar/FollowButton.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import styledComponents from './styledArtist';
+
+const FollowButton = (props) => {
+  const { followStatus } = props;
```

Comment:
This can be destructured in one statement: 
const { followStatus, changeStatus } = props;